minimal-notebook/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM debian:jessie
+
+MAINTAINER Jupyter Project <jupyter@googlegroups.com>
+
+# Install all OS dependencies for fully functional notebook server
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+    git \
+    vim \
+    wget \
+    build-essential \
+    python-dev \
+    ca-certificates \
+    bzip2 \
+    unzip \
+    libsm6 \
+    pandoc \
+    texlive-latex-base \
+    texlive-latex-extra \
+    texlive-fonts-extra \
+    texlive-fonts-recommended \
+    supervisor \
+    sudo \
+    && apt-get clean
+
+ENV CONDA_DIR /opt/conda
+ENV NB_USER jovyan
```

Comment:
Ok, cool, I'm glad you adopted a non-root convention here as well.

We're going to end up getting reports about file permissions during mounting but we'll cross that bridge (again) later.